appengine/src/main/java/com/google/googleinterns/gscribe/modules/Implbindingmodule.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.googleinterns.gscribe.modules;
+
+import com.google.googleinterns.gscribe.services.ExamParserService;
+import com.google.googleinterns.gscribe.services.TokenService;
+import com.google.googleinterns.gscribe.services.impl.ExamParserServiceImpl;
+import com.google.googleinterns.gscribe.services.impl.TokenServiceImpl;
+import com.google.googleinterns.gscribe.utils.*;
+import com.google.googleinterns.gscribe.utils.impl.*;
+import com.google.inject.AbstractModule;
+
+public class Implbindingmodule extends AbstractModule {
```

Comment:
Ideally, it should be divided into multiple modules with each one having it's own responsibility.

Reply:
If Service is the convention, then you can call it ExamGenerationService instead of ExamGenerator. And rename this to ServicesModule.